job-dsl-core/src/test/groovy/javaposse/jobdsl/dsl/views/ListViewSpec.groovy

```diff
@@ -393,6 +393,57 @@ class ListViewSpec<T extends ListView> extends Specification {
         RegexMatchValue.NAME | '.*'
     }
 
+    def 'allJobsFilter'() {
```

Comment:
You do not need to use camel case in Spock test method names. Rename this to `all jobs filter`